test/e2e/cluster.sh

```diff
@@ -102,17 +102,17 @@ if [ -n "${GINKGO_SKIP}" ]; then
     SKIP_AFTER_SCALE_UP="${GINKGO_SKIP}"
   fi
   if [ "${SCALE_CLUSTER}" = "true" ]; then
-    SKIP_AFTER_UPGRADE="${GINKGO_SKIP}|${SKIP_AFTER_SCALE_DOWN}"
+    SKIP_AFTER_UPGRADE="${GINKGO_SKIP}|${SKIP_AFTER_SCALE_DOWN}|${SKIP_AFTER_UPGRADE}"
   else
-    SKIP_AFTER_UPGRADE="${GINKGO_SKIP}"
+    SKIP_AFTER_UPGRADE="${GINKGO_SKIP}|${SKIP_AFTER_UPGRADE}"
```

Comment:
I think this one should not be change

Reply:
My understanding is that the logic here is planned for the scale_down -> upgrade -> scale_up flow, and thus if there is "SCALE_CLUSTER" param (which means scale down & scale operations will be executed), then the upgrade_skip should also include scale_down_skip, otherwise it's just upgrade_skip.